web/ui/src/common/baseFactory.js

```diff
@@ -65,6 +65,9 @@
                     }
 
                     deferred.resolve();
+                })
+                .error((data, status) => {
+                    console.error("Unable to update factory", data);
```

Comment:
Should probably log the `status` as well.

Will the polling continue after an error? If so, we should open a new issue to look into how we want to handle that. The topic is a out of scope for this PR, but if the server returns an error, repeating the same request is probably not going to improve the situation